gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR: Make Prof & Dan normal users instead of superusers and make accounts for students

### Description

I modified the make_users command to make Professor & Dan normal users instead of superusers

### Testing Directions

1. Get the code from this PR
1. Delete db.sqlite3
1. Run `python manage.py migrate && python manage.py make_users`
1. See that accounts are made for Prof & Dan as normal users, along with the usual test accounts, and accounts for each student in the class. This simulates what will happen when we deploy after a DB reset.
![image](https://user-images.githubusercontent.com/13837978/69373894-df470500-0c72-11ea-80bd-839c786398c6.png)
1. Run `python manage.py make_users` again. See that no new users are created because the command realizes that the users already exist. This simulates what will happen when we deploy without a DB reset.
![image](https://user-images.githubusercontent.com/13837978/69373903-e837d680-0c72-11ea-801c-a8185d097122.png)
1. Sign in as Prof with email gcivil@nyu.edu and password heyhey123. See that it works.
1. Sign in as Dan with email dgopstein@nyu.edu and password heyhey123. See that it works.
1. Choose a random student and sign in as them. See that it works.

### Miscellaneous Comments

* We can't truly test Procfile locally because it only applies to Heroku deployments, and the Heroku environment is different from our local ones. But as noted in the testing steps, we can try to simulate what Procfile will do in various scenarios.
* I edited our document with the info about the test users to reflect that Prof & Dan are no longer superusers and also to have the new users for each student
* I got the names and email addresses of the students from the roster on NYU Classes.

### Checklist
